src/apps/material/helper.ts

```diff
@@ -684,6 +685,19 @@ export const getWorkTitle = (work: Work): string => {
   return "Unknown title";
 };
 
+// Constants
+export const MANIFESTATION_HASH_PREFIX = "manifestation-" as const;
+
+export const createManifestationId = (pid: string): string => {
+  return `${MANIFESTATION_HASH_PREFIX}${pid}`;
+};
+
+export const shouldOpenManifestationDetails = (pid: string): boolean => {
```

Comment:
Function name focused on one use case, but it's more generic than that. `pidInUrlHash` would be a better description. But also see next comment.